site/components/wishlist/WishlistCard/WishlistCard.tsx

```diff
@@ -40,7 +42,7 @@ const WishlistCard: React.FC<{
     try {
       // If this action succeeds then there's no need to do `setRemoving(true)`
       // because the component will be removed from the view
-      await removeItem({ id: item.id! })
+      await removeItem({ productId: item.id, variantId: variant })
```

Comment:
why not just keep the current API interface and just remove product with all variants of the same product? This will not be a breaking change

Reply:
If you think this is a breaking change or will fail on other providers (because of the way I designed it) this is not a problem we can modify it to remove every item in wishlist and all its variants.